.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+return (new \PhpCsFixer\Config())
+    ->setFinder(
+        \PhpCsFixer\Finder::create()
+            ->in([
+                getcwd().'/src',
+                getcwd().'/tests',
+            ])
+            ->exclude([
+                'migrations/*',
+            ])
+    )
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PSR12' => true,
+        '@Symfony' => true,
+        'align_multiline_comment' => true,
+        'array_indentation' => true,
+        'blank_line_after_opening_tag' => false, // allows for <?php declare(strict_types=1);
```

Comment:
in our existing SF projects, we got this on a blank line.
No deal breaker for me

Reply:
This is one taken over from @janvernieuwe, 

I have no real preference for this rule, but as the comment suggest `<?php declare(strict_types=1);` 
I can see reason why that one would not need a new line for declaring strict types.

Reply:
It's how I do it, but if there already are multiple projects that enforce this. It is possibly better to set it to true.
I'm not making MY conventions, but the company's :)

Reply:
Aren't those conventions based on what suits the needs of the devs out of experience, 
so @janvernieuwe I would say your personal pref has some say in it :wink: 

Anyway commit: https://github.com/janvernieuwe/conventions/pull/2/commits/8987b2e5b6857c5df8ea38c85161b3e06e72d53d
contains the changes.

Reply:
It's a minor thing anyways, wait until Toon and I start fighting about array alignments....